.gitignore

```diff
@@ -131,4 +131,4 @@ dmypy.json
 .pyre/
 
 # Training outputs
-./bin/train/outputs/
+./train/outputs/
```

Comment:
Don't move this stuff around. Let me handle it.

Reply:
I'll keep everything under the `bin` folder then - I thought of removing it since there was no more a "bin" there, strictly speaking, but sorry about that 👍